apps/fyle/models.py

```
@@ -377,6 +377,8 @@ def create_expense_groups_by_report_id_fund_source(expense_objects: List[Expense
         """
         Group expense by report_id and fund_source
         """
+        expense_groups = []
+        filtered_corporate_credit_card_expense_groups = []
```

Ruff (0.8.2)

Local variable `filtered_corporate_credit_card_expense_groups` is assigned to but never used

Remove assignment to unused variable `filtered_corporate_credit_card_expense_groups`

(F841)